tests/test_compression_lr_multiplier.py

```diff
@@ -0,0 +1,147 @@
+"""
+ Copyright (c) 2019-2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import contextlib
+import copy
+import pytest
+import torch
+import torch.nn as nn
+from torch.optim import SGD
+
+from nncf.layer_utils import CompressionParameter
+from tests.helpers import TwoConvTestModel, create_compressed_model_and_algo_for_test,\
+    get_grads, check_equal, check_not_equal
+from tests.quantization.test_algo_quantization import get_quantization_config_without_range_init
+from tests.sparsity.rb.test_algo import get_basic_sparsity_config
+
+
+@contextlib.contextmanager
+def torch_seed():
+    seed = torch.seed()
+    torch.manual_seed(0)
+    yield
+    torch.manual_seed(seed)
+
+
+def get_quantization_config():
+    config = get_quantization_config_without_range_init()
+    return config
+
+
+def get_sparsity_config():
+    config = get_basic_sparsity_config()
+    return config
+
+
+def modify_config(config, multiplier, multiplier_location, use_algo_list=False):
+    config = copy.deepcopy(config)
+
+    if use_algo_list:
+        config['compression'] = [config['compression']]
+
+    if multiplier_location in ['local', 'both']:
+        if isinstance(config['compression'], list):
+            for algo in config['compression']:
+                algo.update({
+                    'compression_lr_multiplier': multiplier
+                })
+        else:
+            config['compression'].update({
+                'compression_lr_multiplier': multiplier
+            })
+
+    if multiplier_location in ['global', 'both']:
+        config.update({
+            'compression_lr_multiplier': multiplier
+        })
+
+    return config
+
+
+def divide_params(params):
+    regular_params, compression_params = [], []
+    for param in params:
+        if param.requires_grad:
+            if isinstance(param, CompressionParameter):
+                compression_params.append(param)
+            else:
+                regular_params.append(param)
+
+    return regular_params, compression_params
+
+
+def make_train_step(config):
```

Comment:
`get_model_parameters_after_single_optimizer_step`. Setting up type hints in the helper functions is also good for both understanding and debugging.

Reply:
Done.